doc/nrf/releases/release-notes-changelog.rst

```diff
@@ -129,6 +129,7 @@ nRF9160 samples
 
   * Added a new shell command ``cloud`` for establishing an MQTT connection to nRF Cloud.
   * Removed support for the GPS driver.
+  * DK LED #1 is indicating LTE registration status.
```

Comment:
```suggestion
  * The LED1 on the development kit indicates the LTE registration status.
```
Should this be also documented in the actual sample doc?

Reply:
A good point. Added. Please, check.